internetarchive/cli/ia_copy.py

```diff
@@ -97,7 +102,16 @@ def main(argv, session, cmd='copy'):
         sys.exit(1)
 
     args['--header']['x-amz-copy-source'] = '/{}'.format(parse.quote(src_path))
-    args['--header']['x-amz-metadata-directive'] = 'COPY'
+    # Copy the old metadata verbatim if no additional metadata is supplied,
+    # else combine the old and the new metadata in a sensible manner.
+    args['--header']['x-amz-metadata-directive'] = ('REPLACE' if (args['--metadata']
```

Comment:
Could you please use multi-line statements in cases like these? Thanks again!

Reply:
Is the latest commit what you had in mind?

The duplication of `args['--header']['x-amz-metadata-directive']` seems like the most readable form to me.